test/boost/network_topology_strategy_test.cc

```diff
@@ -132,7 +133,8 @@ void endpoints_check(
     }
 
     for (auto&& [dc, rf] : dc_rf) {
```

Comment:
This reminds me: when dropping rf, if a rack is down (or even just a node in the rack), drop that rack preferentially. This allows easier recovery from the complete loss of a rack.

Reply:
I discussed dropping replicas with @tgrabiec and we came to the conclusion that we want to always drop replicas in reverse ordinal order so not to break replica pairing for MVs.

With only a node down in a rack, the base table might decide to drop this replica, while the view places a replica on a live node in that rack and will reach a different solution, so you proposal makes sense only when the whole rack is down.

Maybe we should consider an extension to the network-topology-strategy options so one could specify rf per rack (e.g. `{'dc/rack': 'rf', ...}` and then it could be set to 0 in case the rack is to be removed.

Cc @nyh